cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -144,9 +144,11 @@ screen.badworkstation.heading=You cannot login from this workstation.
 screen.badworkstation.message=Please contact <a style="white-space: nowrap" href="mailto:support@osf.io">support@osf.io</a> to regain access.
 
 # OSF Login Failure Pages
-screen.loginnotallowed.heading=Account not confirmed
-screen.loginnotallowed.message=The OSF account associated with the email has been registered but not confirmed.  Please check your email (and spam folder) or click the button below to resend your confirmation email.
-screen.loginnotallowed.button.resendConfirmation=Resend confirmation email
+screen.accountnotconfirmed.osflogin.heading=Account not confirmed
+screen.accountnotconfirmed.osflogin.message=The OSF account associated with the email has been registered but not confirmed.  Please check your email (and spam folder) or click the button below to resend your confirmation email.
+screen.accountnotconfirmed.osflogin.button.resendConfirmation=Resend confirmation email
+screen.accountnotconfirmed.idplogin.heading=Account not confirmed
+screen.accountnotconfirmed.idplogin.message=The OSF account associated with the email has been registered but not confirmed. Our records show that this account was created via ORCiD login. Please check your email (and spam folder) for the confirmation link. If you believe this should not happen, please contact <a style="white-space: nowrap" href="mailto:support@osf.io">OSF Support</a>.
```

Comment:
Not currently relevant, but it would be nice to template the provider in, rather than having it always say ORCiD

Reply:
Yep.

* We need to fix an issue where CAS swallows exception details and turns them into general authentication exceptions before we can template providers. See https://openscience.atlassian.net/browse/ENG-1185

* In addition, although I use "external idp" or "idp login" for class and var names,  I decided to put ORCiD directly in the messages since it is the only one and we don't make things complicated for the user.

* Finally, a screenshot of the error pages.
![account-not-confirmed-osf](https://user-images.githubusercontent.com/3750414/73683569-79bebe00-4690-11ea-897a-7c728506db91.png)

![account-not-confirmed-idp](https://user-images.githubusercontent.com/3750414/73683570-79bebe00-4690-11ea-9af7-851249cea435.png)